src/commands/moderation/lockdown.ts

```diff
@@ -18,6 +19,10 @@ export default class implements Command {
 		await interaction.deferReply({ ephemeral: true });
 		await checkModRole(interaction, locale);
 
+		const {
+			client: { user: clientUser },
+		} = interaction;
```

Comment:
If you are really only using that once below, might as well use the "long" form, imo.